Doctrine/Phpcr/Form/Type/StaticContentType.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr\Form\Type;
+
+use Doctrine\Bundle\PHPCRBundle\Form\DataTransformer\DocumentToPathTransformer;
+use Doctrine\ODM\PHPCR\DocumentManager;
+use Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr\StaticContent;
+use Symfony\Cmf\Bundle\ContentBundle\Form\Type\StaticContentType as ModelStaticContentType;
+use Symfony\Cmf\Bundle\TreeBrowserBundle\Form\Type\TreeSelectType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class StaticContentType extends AbstractType
+{
+    /**
+     * @var DocumentManager
+     */
+    private $documentManager;
+
+    /**
+     * @var string
+     */
+    private $contentBasePath;
+
+    /**
+     * @param DocumentManager $documentManager
+     * @param string          $contentBasePath
+     */
+    public function __construct(DocumentManager $documentManager, $contentBasePath)
+    {
+        $this->documentManager = $documentManager;
+        $this->contentBasePath = $contentBasePath;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
```

Comment:
we should document the supported values in $options. we should also use the OptionsResolver to make sure the options are not all required. we can then also validate those options to only have valid values. this will make it easier to discover mistakes with the form builder.